site/en/docs/extensions/support-feedback/file-a-bug/index.md

```diff
@@ -9,7 +9,7 @@ tags:
 
 ## File a bug
 
-While developing an extension, you may find behavior that does not match the extension's documentation or is otherwise unexpected. This may be the result of a Chrome bug. Regardless, please let us know by filing an appropriate issue report. Please provide enough information to reproduce the issue by following these steps:
+While developing an extension, you may find behavior that does not match the extension's documentation or is otherwise unexpected. This may be the result of a Chrome bug, or it may be something we should add to documentation. Regardless, please let us know by filing an appropriate issue report. Please provide enough information to reproduce the issue by following these steps:
```

Comment:
```suggestion
While developing an extension, you may find behavior that does not match the extension's documentation or is otherwise unexpected. This may be the result of a Chrome bug or something we should add to the documentation. Regardless, please let us know by filing an appropriate issue report. Please provide enough information to reproduce the issue by following these steps:
```